### PR TITLE
Use print() function in both Python 2 and Python 3

### PR DESCRIPTION
**Pull request by cclauss**
Legacy __print__ statements are syntax errors in Python 3 but __print()__ function works as expected in both Python 2 and Python 3.

### CODE REVIEW

**Comment by AlCutter** [comment]
Thank you so much for doing this, Christian!

**Review by betadae — COMMENTED** [review]
(234567)For Africa

File: python/utilities/precerts_finder/precerts_finder.py [hunk 1]
```diff
@@ -44,8 +45,8 @@ def run():
         write_matched_certificate,
         start_entry=FLAGS.start_entry)
 
-    print "Scanned %d, %d matched and %d failed strict or partial parsing" % (
-        res.total, res.matches, res.errors)
+    print("Scanned %d, %d matched and %d failed strict or partial parsing" % (
+        res.total, res.matches, res.errors))
```
Comment (betadae) [inline python/utilities/precerts_finder/precerts_finder.py:51]: Basit Iddrisu

**Review by betadae — COMMENTED** [review]
Abdul Basit Iddrisu

**Pull request merged**